cpp/CMakeLists.txt

```diff
@@ -319,6 +322,8 @@ if(RAFT_COMPILE_DIST_LIBRARY)
           )
   target_compile_definitions(raft_distance_lib
           INTERFACE "RAFT_DISTANCE_COMPILED")
+  target_compile_definitions(raft_distance_lib
+          INTERFACE "CUTLASS_NAMESPACE=${RAFT_CUTLASS_NAMESPACE}")
```

Comment:
This won't work as you think. The `RAFT_CUTLASS_NAMESPACE` variable doesn't exist at this scope. Two instead you should link to the `nvidia::cutlass::cutlass` target which will automatically bring this in.